src/settings/collectionsettingspage.cpp

```diff
@@ -130,6 +145,20 @@ void CollectionSettingsPage::Load() {
   ui_->checkbox_cover_lowercase->setChecked(s.value("cover_lowercase", true).toBool());
   ui_->checkbox_cover_replace_spaces->setChecked(s.value("cover_replace_spaces", true).toBool());
 
+  ui_->spinbox_cache_size->setValue(s.value(kSettingsCacheSize, 80).toInt());
+  ui_->combobox_cache_size->addItems(cacheUnitNames);
+  ui_->combobox_cache_size->setCurrentIndex(s.value(kSettingsCacheSizeUnit, (int) CacheSizeUnit_MB).toInt());
+  ui_->checkbox_disk_cache->setChecked(s.value(kSettingsDiskCacheEnable, false).toBool());
+  ui_->label_disk_cache_size->setEnabled(ui_->checkbox_disk_cache->isChecked());
+  ui_->spinbox_disk_cache_size->setEnabled(ui_->checkbox_disk_cache->isChecked());
+  ui_->spinbox_disk_cache_size->setValue(s.value(kSettingsDiskCacheSize, 80).toInt());
+  ui_->combobox_disk_cache_size->setEnabled(ui_->checkbox_disk_cache->isChecked());
+  ui_->combobox_disk_cache_size->addItems(cacheUnitNames);
+  ui_->combobox_disk_cache_size->setCurrentIndex(s.value(kSettingsDiskCacheSizeUnit, (int) CacheSizeUnit_MB).toInt());
+
+  connect(ui_->checkbox_disk_cache, SIGNAL(stateChanged(int)), SLOT(DiskCacheEnable(int)));
+  connect(ui_->button_disk_cache, SIGNAL(clicked()), SLOT(dialog->app()->ClearPixmapDiskCacheSlot()));
```

Comment:
This connect is incorrect, you probably mean: connect(ui_->button_disk_cache, SIGNAL(clicked()), dialog()->app(), SLOT(ClearPixmapDiskCacheSlot()));
But there is no need for a slot at all here. Just connect from signal to signal.
ie: connect(ui_->button_disk_cache, SIGNAL(clicked()), dialog()->app(), SIGNAL(ClearPixmapDiskCache()));